introduction.md

```diff
@@ -0,0 +1,6 @@
+My name is Silvestre Vivo. This is an example from CodeSchool to learn how Git Hub works.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.